src/index.html

```diff
@@ -15,8 +15,108 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <title>Moyo header</title>
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#header"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo-Logo"
+        />
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol. Also, ensure that the `href` attribute links to the correct section of the page. If `#header` is not an actual id on the page, it should be updated to reflect the correct navigation target.